airbyte-webapp/src/components/ui/DropdownMenu/DropdownMenu.tsx

```diff
@@ -72,11 +72,17 @@ export const DropdownMenu: React.FC<React.PropsWithChildren<DropdownMenuProps>>
             {options.map((item, index) => (
               <Menu.Item key={index}>
                 {({ active }) =>
-                  React.createElement(
-                    item.as ?? "button",
-                    { ...elementProps(item, active) },
-                    <MenuItemContent data={item} />
-                  )
+                  item.as === "a"
+                    ? React.createElement(
+                        item.internal ? Link : "a",
```

Comment:
Using a `Link` allows for `internal` links to just render the new page within the MainView, rather than reload the entire tab.